src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -80,7 +80,7 @@ public void GetRequestResponseAfterAbort_Throws()
         [Fact]
         public void NotImplementedMembers_Throws()
         {
-            WebRequest request = WebRequest.Create("file://anything");
+            WebRequest request = WebRequest.Create("file:///anything");
```

Comment:
I'll revert the change of unix paths having to start with '/'. (For implicit paths, this is the case still.) But for file:// uris, a backslash will do to. The Uri class will convert these backslashes to forwardslashes.
e.g.:
```
/one\two ->          path = /one\two
file://\one\two ->   path = /one/two
file:///one%5Ctwo -> path = /one\two
```